bika/lims/content/analysisrequest.py

```diff
@@ -1369,6 +1370,22 @@
             visible=False,
         ),
     ),
+    BooleanField(
+        "InternalUse",
+        mode="rw",
+        required=0,
+        default=False,
+        read_permission=View,
+        write_permission=FieldEditInternalUse,
+        widget=BooleanWidget(
+            label=_("Internal use"),
+            description=_("This sample is only for internal use. Won't be "
+                          "displayed to client"),
```

Comment:
Would you mind to change the description to something more descriptive, e.g.:
Mark the sample for internal use only. This means it is only accessible to lab personell and not to clients.